bio_tools/bio_tools.py

```diff
@@ -0,0 +1,134 @@
+from modules import dna_rna_tools_modules as drtm
+from modules import filter_fastq_modules as ffm
+
+"""
+The module provides functions for processing
+DNA/RNA sequences and filtering FASTQ data.
+
+Functions:
+- run_dna_rna_tools: Performs operations on DNA/RNA sequences.
+- filter_fastq: Filters FASTQ files by specified
+parameters of quality, length and GC content.
+
+Author: Nika Samusik
+"""
+
+
+def run_dna_rna_tools(*args) -> list | str:
+    """
+    Performs an operation on DNA or RNA sequences
+from the drtm module
+
+    Parameters
+    ----------
+    *args : tuple, str
+        DNA/RNA sequences and an element with the operation name.
+        The operation must be passed as the last argument.
+
+    Returns
+    -------
+    answer : list, str
+        Results or result of executing functions from the module
+        drtm. Invalid input warning
+        if the submitted data does not contain an operation or
+        submits an invalid operation
+
+    Raises
+    -------
+    ValueError
+        if the sequences are not DNA/RNA or
+    KeyError
+        if the operation is not supported
+    """
+    if len(args) == 1:
```

Comment:
можно обобщить условие и написать :( чтобы вообще удостовериться что у нас есть последовательность)
```suggestion
    if len(args) < 2:
```